server/src/controller/order.controller.ts

```diff
@@ -1,53 +1,68 @@
 import { Response, Request } from "express" 
-import db from '../db'
 import { transporter } from '../services/nodemailer'
 import { v4 as uuidv4 } from 'uuid'
 import { RESOURCE } from "../../data/constants/routeConstants"
+import { Order, Clock, User, Master, City } from '../models/Models'
 
 
 export const postOrder = async (req: Request, res: Response) => {
         
     try {
         const { clock_id, city_id, master_id, start_work_on, name, email } = req.body
-        
-        const durationTime = await db.query('SELECT installation_time FROM clocks WHERE id = $1',[clock_id])
 
-        let user_id
-        const userId = await db.query('SELECT id FROM users WHERE email = $1', [email])
+        const [createdUser, isUserCreated] = await User.findOrCreate({
+            where: {
+                email: email
+            }, 
+            defaults: {
+                email: email,
+                name: name
+            }
 
-        if(userId.rows.length) {
-            user_id = userId.rows[0].id
-        }
+        })
 
-        if(!userId.rows.length) {
-            const createUser = await db.query('INSERT INTO users (name, email) VALUES ($1, $2) RETURNING *', [name, email])
-            user_id = createUser.rows[0].id
-        }
+        const userId = createdUser.id
 
-        if(durationTime.rows.length) {
-            const { installation_time } = durationTime.rows[0]
-            
-            let date = new Date(`${start_work_on}`)
+                const durationTime = await Clock.findOne({
+                    attributes: ['installation_time'],
+                    where: {
+                        id: clock_id
+                    }
+                })
 
-            date.setUTCHours(date.getHours() + installation_time)
+        if(durationTime) {
+            const { installation_time } = durationTime
             
-            const end_work_on = date.toISOString()
+            let endDate = new Date(`${start_work_on}`)
+            let startDate = new Date(`${start_work_on}`)
+            startDate.setUTCHours(startDate.getHours())
+            endDate.setUTCHours(endDate.getHours() + installation_time)
+            const endWorkOn = endDate.toISOString()
+            const startWorkOn = startDate.toISOString()
 
             const ratingIdentificator = uuidv4();
-
-            const createOrder = await db.query(`INSERT INTO orders (clocks_id, user_id, city_id, master_id, start_work_on, end_work_on, uuid_id) 
-                                                        VALUES ($1, $2, $3, $4, $5, $6, $7)`, [clock_id, user_id, city_id, master_id, start_work_on, end_work_on, ratingIdentificator])
+            console.log(clock_id, userId, city_id, master_id, startWorkOn, endWorkOn, ratingIdentificator);
```

Comment:
Do not push console logs. Doesn't matter it's a client or server. Only if you have a logger use it.